pegasus/test/test_i18n_hoc_routes.rb

```diff
@@ -22,6 +23,14 @@ def test_hoc_pages_for_all_locales
     end
   end
 
+  def test_hoc_partner_pages
+    CDO.partners.each do |partner|
+      header 'Host', "#{partner}.code.org"
+      resp = get('/')
+      assert_equal 200, resp.status, "#{partner}/"
+    end
+  end
```

Comment:
:+1: I'm also updating our UI/Eyes tests to actually check these on test, rather than production, in https://github.com/code-dot-org/code-dot-org/pull/21031.